src/lib/chips/chip.ts

```diff
@@ -167,11 +170,13 @@ export class MatChip extends _MatChipMixinBase implements FocusableOption, OnDes
   protected _value: any;
 
   /**
-   * Whether or not the chips are selectable. When a chip is not selectable,
-   * changes to it's selected state are always ignored.
+   * Whether or not the chip is selectable. When a chip is not selectable,
+   * changes to it's selected state are always ignored. By default a chip is
+   * selectable, and it becomes non-selectable if it's parent chip list is
+   * not selectable.
    */
   @Input()
-  get selectable(): boolean { return this._selectable; }
+  get selectable(): boolean { return this._selectable && this.chipListSelectable; }
```

Comment:
Does this work with the `OnPush` change detection? I think the chip may need a `markForCheck` somewhere.

Reply:
The `selectable` on chip list is an `@Input()`.